monitoring/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+appVersion: "1.0"
+description: "A combination of the community Prometheus and Helm charts."
```

Comment:
Is this supposed to be "Prometheus and Grafana charts"?

Reply:
Sure is. Corrected.